runtime/arc.js

```diff
@@ -55,7 +53,7 @@ class Arc {
   }
 
   clone() {
-    var arc = new Arc({loader: this._loader, id: this.generateID(), pecFactory: this._pecFactory, slotManager: this._slotManager});
+    var arc = new Arc({loader: this._loader, id: this.generateID(), pecFactory: this._pecFactory, slotManager: this.pec.slotManager});
```

Comment:
I wasn't sure - currently the clone() method is only used from the speculator, so it'll be fine.
But is the clone() method only ever intended to be called from the speculator?